golem-base-indexer/golem-base-indexer-server/src/server.rs

```diff
@@ -61,6 +61,8 @@ pub async fn run(
     let health = Arc::new(HealthService::default());
 
     // TODO: init services here
+    let leaderboards_updater = LeaderboardsUpdaterService::new(Arc::clone(&db_connection));
+    leaderboards_updater.spawn_periodic_task(1800); // 1800 seconds = 30 minutes
```

Comment:
Let's spawn it in indexer.rs, where we already spawn a task for refreshing metrics: https://github.com/Golem-Base/blockscout-rs-neti/blob/main/golem-base-indexer/golem-base-indexer-server/src/indexer.rs#L13